docs/en/api/POSIX-API.md

```diff
@@ -425,6 +426,95 @@ Data can be cached on ramdisk or disk based on the type of the cache directory.
   {% endnavtab %}
 {% endnavtabs %}
 
+### Security Configuration
+
+The security of the Alluxio POSIX API does not exactly follow the POSIX standard.
+This is a known limitation and we are working to improve it.
+
+#### Permission Check
+
+All file/dir permissions in Alluxio POSIX API are checked against the user launching the AlluxioFuse process instead of the end user running the operations.
+
+#### User Group Policy
+
+User group policies decide the user/group of the created file/dir and the user/group shown in the get file/dir path status operations.
+
+Three user group policies can be chosen and configured via setting `alluxio.fuse.auth.policy.class=<policy_class_name>`
+in `${ALLUXIO_HOME}/conf/alluxio-site.properties`.
+
+- Default launch user group policy (`alluxio.fuse.auth.policy.class=alluxio.fuse.auth.LaunchUserGroupAuthPolicy`):
+Assuming user `alluxio-user` with group `alluxio-group` launches the FUSE process.
+```console
+# The user/group of all files/dirs created through the FUSE mount point is set to the user/group that launches the FUSE application
+$ touch /mnt/people/file
+$ ls -al /mnt/people/file
+-rw-r--r--    1 alluxio-user  alluxio-group  0 Oct 11 23:26 file
+$ ${ALLUXIO_HOME}/bin/alluxio fs ls /people/file
+-rw-r--r--  alluxio-user  alluxio-group  0  PERSISTED 10-11-2022 23:26:03:406 100% /people/file
+
+# The user/group of all files/dirs is shown as the user/group that launches the FUSE application regardless of the actual user/group in the underlying Alluxio system
```

Comment:
I'm a little confused here. I believe you are still talking about the files created through Fuse mount point but you use "all files/dirs" so I was thinking about other files.
Maybe just put `Regardless of the actual ...`, and use a different file for the example, so that user would know they are talking about the same point. Just my suggestion, maybe there's a better way

Reply:
Good suggestion. Iterated, PTAL